frontend/common/kubeflow-common-lib/projects/kubeflow/src/styles/styles.scss

```diff
@@ -86,9 +86,7 @@ $kubeflow-theme: mat.define-light-theme(
 }
 
 // Use the material icons offline
-// AAW: Leaving this blank was the way to make this work
-// after angular version update without returning errors
-$material-icons-font-path: '';
+$material-icons-font-path: '~material-icons/iconfont/';
```

Comment:
Wow I swear this used to break things somewhere. But this doesnt seem to break anything so that is good.